packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -375,8 +422,20 @@ async function callDestinationService(
     headers
   };
 
+  let resilience = [
+    timeout<fpp, HttpMiddlewareContext>(),
+    circuitBreakerHttp<fpp, HttpMiddlewareContext>()
+  ];
+  // Do a retry only for the get destination by Name calls which are somethimes flaky for the token retrieval
+  if (options?.destinationName && options.retry) {
+    resilience = [
```

Comment:
We can replace this condition part when we have `resilience()` method.

Reply:
yes nice.